lib/composables/Lang.ts

```diff
@@ -26,7 +26,10 @@ export function useSetupLang(): (user?: HasLang | null) => void {
 }
 
 export function provideLang(lang: Lang) {
-  provide(SefirotLangKey, lang)
+  const instance = getCurrentInstance()
+  if (instance) {
+    instance.appContext.provides[SefirotLangKey] = lang
+  }
```

Comment:
we can add some warning in else, but it always will be available, condition is just a formality